flink-connectors/flink-connector-cassandra/pom.xml

```diff
@@ -61,7 +63,8 @@ under the License.
 							<promoteTransitiveDependencies>true</promoteTransitiveDependencies>
 							<artifactSet>
 								<includes>
-									<include>com.datastax.cassandra:cassandra-driver-core</include>
+									<!--driver was relocated by datastax-->
```

Comment:
```suggestion
```

Reply:
reverting this change as the relocation starts with driver 4.0. I'll do all the deps cleaning, create the user jar and test it on a flink pipeline to check that whole deps and shading are ok.

Reply:
tested on a local flink cluster with packaged user jar and it works.